### DISCUSSION TITLE
4 channel mode (ABCD)

### DISCUSSION THREAD

**Discussion by OE1MWW** [jvde-github/AIS-catcher#151] [General]
Want to track channels AB and CD

On my RTL-SDR dongle, the command line is:
AIS-catcher -N LAT 48.268991 LON 16.384977 SHARE_LOC on -o 0 -v 10 -u 127.0.0.1 29421 -N 8100 PLUGIN_DIR plugins -gr RTLAGC on TUNER auto -a 192K -c X

whatever additional sample rate I set at the end of this command like, ('-s xxxK'), 
"Error: Model: sample rate must be between 12k and 192k (inclusive)."

Thanks for any input ;-)

**Comment by jvde-github** [comment]
Hi, thanks for reaching out. The `-c X` option only listens to one channel and only useful for reading from a file and requires a particular sample rate.

If you want to receive channel A & B, use `-c AB`. If you want to listen to channel C & D, use `-c CD`. Unfortunately, with a RTL-SDR the bandwidth is not sufficient to receive all four channels simultaneous. So you need to use two sticks for that one listening to A/B and the other for C/D.

Does that help?

Jasper

**Comment by OE1MWW** [comment]
Hi Jasper,
thanks for the fast reply. In AISRecWinFull (https://sites.google.com/site/feverlaysoft/home)
is an option for the RTL-SDR to select 'ABCD', so I was in the believe, AIS-catcher may do the same.
Wolfgang

**Comment by jvde-github** [comment]
Hi Wolfgang,

You can try but physically this is not possible. AIS A/B sits at ~162 MHz and C/D at 156.8 Mhz. So that means with one dongle you need to have a sample rate of at least  5.2 MHz (162-156.8) and tune to a frequency in the middle. 

I have tried with an AirSpy at 6 MHz and this was already not great. It worked better with the R2 at 10 MHz. I think that was discussed here: https://github.com/jvde-github/AIS-catcher/issues/77#issuecomment-1475215846

I don't see how AISrec can do this with an RTL-SDR. I have the idea that you can configure ABCD but when you press start it only mentions A/B so ignores CD.

Cheers, Jasper
